CMakeLists.txt

```diff
@@ -189,7 +189,6 @@ install(
 )
 install(
     FILES
-        "${CMAKE_CURRENT_SOURCE_DIR}/DISCLAIMER"
```

Comment:
Might be a good idea to keep this file, and use it to point out the disclaimer regarding MADlib's trademark. This is coming from the fact that MADlib's trademark does not belong to ASF yet, and it was suggested we make this clear in both our homepage and releases.
Tagging @rvs for more on this.